src/runtime/components/data/Table.vue

```diff
@@ -276,7 +276,25 @@ export default defineComponent({
       }
     })
 
-    const indeterminate = computed(() => selected.value && selected.value.length > 0 && selected.value.length < props.rows.length)
+
+    const getStringifiedSet = <T extends unknown>(arr: T[]) => new Set(arr.map(item => JSON.stringify(item)))
+
+    const totalRows = props.rows.length
```

Comment:
What if the `props.rows` change here?

Reply:
i think the totalRows still same as initial render if the rows is change. i should wrap it using computed thank you @benjamincanac